Update update-contao.en.md

fiedsch opened the pull request:
* add english traslation of #578
* revise the automatic translation

TODO:
* localised screen shots

fiedsch committed:
Update update-contao.en.md

* add englich traslation of #578
* revise the automatic translation

TODO:
* localised screen shots

fkaminski commented:
@fiedsch
Maybe you can add the following hints from @fritzmg  (which he had left in my PR):

_Generally I only noticed just now, that the commands use composer.phar. However, it is not explained that you first have to either download the latest composer.phar - or install it globally (and if you installed it globally, you can simply use composer, without prepending php - unless in specific circumstances)._

And maybe a add notice that this procedure basically also works for new installations also...

fiedsch commented:
@fkaminski Thanks. I will include that in my PR (and also the already published german version). 

My original intention was to explain that with

> Composer (wir gehen hier davon aus, dass du Composer in deinem Arbeitsverzeichnis installierst)

which would make `composer.phar` available in the working directory and the examples should work.

I suggest, I add a link to https://docs.contao.org/manual/de/installation/contao-installieren/#composer-installieren
(or https://docs.contao.org/manual/en/installation/install-contao/#install-composer respectively) to make that clearer.

@fritzmg what do you think?

I just see, that @fritzmg's comment also refers to the section https://docs.contao.org/manual/de/installation/contao-aktualisieren/#aktualisierung-ueber-die-kommandozeile (on the server, that is).

fritzmg commented:
Yeah sounds good :)

fiedsch committed:
add new `note` on how to install composer

ermove code example with---most likely---outdated hash value, as this can be found on the linked page anyway

fiedsch committed:
Add note on requirement of having composer installed

* use the "composer installed globally in the examples
* add command prompt `$` in the examples like we do elsewhere

fiedsch commented:
Updated PR. If we agree that this description is what we want, I'd
* create the english screenshots 
* update the german documentation accordingly

fritzmg commented:
_You don't have to_, but while you are at it, could you also review and overhaul the rest of the content of these articles, since they are still only machine translated? 😬 😁

fritzmg commented:
> If we agree that this description is what we want

Imho description looks good :)

xchs commented on docs/manual/installation/install-contao.en.md:
```diff
@@ -52,16 +52,19 @@ cd www
 
-To install Composer, please follow the instructions on the [Composer website and](https://getcomposer.org/download/) copy the commands from there, because the SHA hash used to verify the download changes with each version of Composer. At this point, the commands look like this.
+{{% notice note %}}
+You can either install composer [locally](https://getcomposer.org/doc/00-intro.md#locally) 
```
```suggestion
You can either install Composer [locally](https://getcomposer.org/doc/00-intro.md#locally) 
```

xchs commented on docs/manual/installation/install-contao.en.md:
```diff
@@ -52,16 +52,19 @@ cd www
+or [globally](https://getcomposer.org/doc/00-intro.md#globally). 
+ 
+ If you install locally, you will have a `composer.phar` file in your working directory (i.e. where your project's
+ `composer.json`and `composer.lock` files are located). You would then call composer with `php composer.phar`.
```
```suggestion
 `composer.json`and `composer.lock` files are located). You would then call Composer with `php composer.phar`.
```

xchs commented on docs/manual/installation/update-contao.en.md:
```diff
@@ -76,6 +76,10 @@ Your Contao installation is now up to date.
 ### Updating via the command line {#update-via-the-command-line}
 
+{{% notice note %}}
+ To update Contao via the command line you need to have composer [installed](../install-contao/#install-composer). 
```
```suggestion
 To update Contao via the command line you need to have Composer [installed](../install-contao/#install-composer). 
```

xchs commented on docs/manual/installation/update-contao.en.md:
```diff
@@ -152,7 +156,7 @@ What do you need on your computer?
 - any directory you work in (your working directory)
 - PHP, ideally in the same version as used on your hosting
-- Composer (we assume here that you install composer in your working directory)
+- Composer (we assume here that you [install](../install-contao/#install-composer) composer globally)
```
```suggestion
- Composer (we assume here that you [install](../install-contao/#install-composer) Composer globally)
```

fiedsch committed:
Update docs/manual/installation/install-contao.en.md

Co-authored-by: Hannes <github@gmx.net>

fiedsch committed:
Update docs/manual/installation/install-contao.en.md

Co-authored-by: Hannes <github@gmx.net>

fiedsch committed:
Update docs/manual/installation/update-contao.en.md

Co-authored-by: Hannes <github@gmx.net>

fiedsch committed:
Update docs/manual/installation/update-contao.en.md

Co-authored-by: Hannes <github@gmx.net>

fiedsch commented:
thanks, @xchs!

@fritzmg:
> You don't have to, but while you are at it, could you also review and overhaul the rest of the content of these articles, since they are still only machine translated?

I suggest, I finish this PR (including the respective changes in the german version) and then work on the mashine translated texts in a different PR. Otherwise I fear, I‘m losing track.

fiedsch committed:
"finisch" PR

* add english versions of the new screen shots
*  apply recent changes in the en version to the de version

fiedsch commented:
"finished" PR and removed draft status.

netzarbeiter commented on docs/manual/installation/install-contao.de.md:
```diff
@@ -71,16 +71,16 @@ cd www
+Du kannst Composer entweder [lokal](https://getcomposer.org/doc/00-intro.md#locally) 
+oder [global](https://getcomposer.org/doc/00-intro.md#globally) installieren. 
+ 
+Wenn du Composer lokal installierst, befindet sich die Datei `composer.phar` in deinem Arbeitsverzeichnis (d.h. dort, wo
```
```suggestion
Wenn du Composer lokal installierst, befindet sich die Datei `composer.phar` in deinem Arbeitsverzeichnis (d. h. dort, wo
```

netzarbeiter commented on docs/manual/installation/install-contao.en.md:
```diff
@@ -52,16 +52,19 @@ cd www
+You can either install Composer [locally](https://getcomposer.org/doc/00-intro.md#locally) 
+or [globally](https://getcomposer.org/doc/00-intro.md#globally). 
+ 
+If you install locally, you will have a `composer.phar` file in your working directory (i.e. where your project's
```
Suggestion:
If you install Composer …
If you install it …

netzarbeiter commented on docs/manual/installation/install-contao.en.md:
```diff
@@ -52,16 +52,19 @@ cd www
+If you install locally, you will have a `composer.phar` file in your working directory (i.e. where your project's
+`composer.json` and `composer.lock` files are located). You would then call Composer with `php composer.phar`.
+ 
+If you install globally, you can use the `composer` command in any directory. 
```
Suggestion:
If you install Composer …
If you install it …

netzarbeiter commented on docs/manual/installation/update-contao.en.md:
```diff
@@ -6,89 +6,115 @@ aliases:
-With version 2.11, the [release cycle of Contao](https://contao.org/de/release-plan.html) has been adjusted and Long-Term Support (LTS) versions have been introduced that are supported and updated for 24 months, even if newer Contao versions have been released in the meantime. An overview of all Contao versions is available on Wikipedia.
+With version 2.11, the [release cycle of Contao](https://contao.org/de/release-plan.html) has been adjusted and 
+Long-Term Support (LTS) versions have been introduced that are supported and updated for 24 months, even if newer Contao 
+versions have been released in the meantime. An overview of all Contao versions is available on Wikipedia.
```
```suggestion
versions have been released in the meantime. An overview of all Contao versions is available on [Wikipedia](https://en.wikipedia.org/wiki/Contao#Versions).
```

netzarbeiter commented on docs/manual/installation/update-contao.en.md:
````diff
@@ -98,19 +124,128 @@ If you are updating for a [minor release](#minor-release), you need to install t
+make the suggested changes and then close the install tool.
+
+Instead of the Contao install tool, (with Contao 4.9 or later) you can use the command line and run the command 
+```bash
````
```suggestion

```bash
```

netzarbeiter commented on docs/manual/installation/update-contao.en.md:
```diff
@@ -98,19 +124,128 @@ If you are updating for a [minor release](#minor-release), you need to install t
+You need a local Contao installation. In this installation, you install the Contao Manager and run the update like
+described in the section [Updating with the Contao Manager](#updating-with-the-contao-manager). 
+
+Before doing so, make sure that the Composer Resolver Cloud is not used! You do not need it, because you you can provide 
```
```suggestion
Before doing so, make sure that the Composer Resolver Cloud is not used! You don't need it, because you can provide 
```

netzarbeiter reviewed (changes requested):
Thank you, Andreas.

fiedsch committed:
Update docs/manual/installation/install-contao.de.md

Co-authored-by: netzarbeiter <netzarbeiter@users.noreply.github.com>

fiedsch committed:
Update docs/manual/installation/update-contao.en.md

Co-authored-by: netzarbeiter <netzarbeiter@users.noreply.github.com>

fiedsch committed:
Update docs/manual/installation/update-contao.en.md

Co-authored-by: netzarbeiter <netzarbeiter@users.noreply.github.com>

fiedsch committed:
Update docs/manual/installation/update-contao.en.md

Co-authored-by: netzarbeiter <netzarbeiter@users.noreply.github.com>

fiedsch committed:
apply @netzarbeiter's suggestions

fiedsch commented:
Thank you @netzarbeiter. I updated the PR.
